htdocs/accountancy/admin/fiscalyear_card.php

```diff
@@ -205,8 +205,8 @@
 			
 			// Ref
 			print "<tr>";
-			print '<td width="20%">' . $langs->trans("Ref") . '</td><td>';
-			print $object->rowid;
+			print '<td width="25%">' . $langs->trans("Ref") . '</td><td>';
```

Comment:
With 4.0, it is recommended now to use hier
print '<td class="titlefield"
or
print '<td class="titlefieldcreate" for create or edit form